api/app/weather_models/machine_learning.py

```diff
@@ -136,7 +136,8 @@ def _add_sample_to_collection(self,
                                   actual: HourlyActual,
                                   sample_collection: SampleCollection):
         """ Take the provided prediction and observed value, adding them to the collection of samples """
-        for model_key, sample_key in zip(MODEL_VALUE_KEYS, SAMPLE_VALUE_KEYS):
+        # TODO: add precip and wind speed/direction to SAMPLE_VALUE_KEYS
```

Comment:
Yep - right now we're only doing temp/rh - and there's no-one I'm aware of asking for adding anything else. It would have to happen in a different ticket.